core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -261,13 +264,14 @@ public int getValue() {
   boolean exists(String path) throws IOException;
 
   /**
-   * Gets the access control list of a file or directory in under file system.
+   * Gets the ACL and the Default ACL of a file or directory in under file system.
    *
    * @param path the path to the file or directory
-   * @return the access control list, or null if ACL is unsupported or disabled
+   * @return the access control list, along with a Default ACL if it is a directory
```

Comment:
hrmmm, I think we really need to consider combining the default acls into the acls object. They really seem like a single unit to me.